CHANGES.rst

```diff
@@ -345,6 +345,8 @@ Bug Fixes
     float32 instead of float64 (with the E and F formats). These values are now
     always read as float64. [#5362]
 
+  - Fixed memoryleak when using the compression module. [#5399, 5464]
```

Comment:
I haven't looked into the code, but I think our favourite astrobot checks the issue numbers and needs the ``#``